File: cfgs/java_cfgs_foo/SQUARE_ROOT_OF_A_PERFECT_SQUARE.java
Nodes
1;special;;;
2;statement;float x = foo_1;14;
3;statement;float y = 1;15;
4;statement;double e = 1.0E-6;16;
5;conditional;(foo_1 - foo_2) > foo_3;17;
6;statement;x = (foo_1 + foo_2) / 2;18;
7;statement;y = foo_1 / foo_2;19;
8;statement;return foo_1;21;
9;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;8;6;
5;7;
6;5;
7;9;
8;

File: cfgs/pythoncfgs/SQUARE_ROOT_OF_A_PERFECT_SQUARE.py
Nodes
1;statement;foo_1 = foo_2;8;
2;statement;foo_1 = 1;9;
3;statement;foo_1 = 1e-06;10;
4;conditional;foo_1 - foo_2 > foo_3;11;
5;statement;return foo_1;14;
6;statement;foo_1 = (foo_2 + foo_3) / 2;12;
7;statement;foo_1 = foo_2 / foo_3;13;
8;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;8;
5;7;
6;4;
7;
